documentation/domains/Cluster.md

```diff
@@ -56,7 +56,7 @@ The specification of the operation of the WebLogic cluster. Required.
 | `affinity` | [Affinity](k8s1.13.5.md#affinity) | The Pod's scheduling constraints. More info: https://oracle.github.io/weblogic-kubernetes-operator/faq/node-heating/.  See `kubectl explain pods.spec.affinity`. |
 | `annotations` | Map | The annotations to be added to generated resources. |
 | `containers` | Array of [Container](k8s1.13.5.md#container) | Additional containers to be included in the server Pod. See `kubectl explain pods.spec.containers`. |
-| `containerSecurityContext` | [Security Context](k8s1.13.5.md#security-context) | Container-level security attributes. Will override any matching Pod-level attributes. See `kubectl explain pods.spec.containers.securityContext`. |
+| `containerSecurityContext` | [Security Context](k8s1.13.5.md#security-context) | Container-level security attributes. Will override any matching Pod-level attributes. See `kubectl explain pods.spec.containers.securityContext`. Beginning with operator version 4.0.5, the operator will use default content for container-level `securityContext` if no value is specified for this field. More info: https://oracle.github.io/weblogic-kubernetes-operator/security/domain-security/pod-and-container/ |
```

Comment:
-> Beginning with operator version 4.0.5, if no value is specified for this field, the operator will use default content for container-level `securityContext`.
-> Period at the end of the last sentence.